memory-exhaustion/memory-exhaustion

```diff
@@ -216,7 +216,7 @@ handle_error() {
 
 memory_exhaustion_test() {
     # Constants
-    local TIMEOUT=1500  # 25 minutes in seconds
+    local TIMEOUT="$MEMORY_EXHAUSTION_TEST_TIMEOUT"
```

Comment:
consider this 
TIMEOUT="${MEMORY_EXHAUSTION_TEST_TIMEOUT:-1500}"

Reply:
thanks~ done